### DISCUSSION TITLE
[BUG] Broken capcha on Qameta Software - Help Desk (Sign up for support portal)

### DISCUSSION THREAD

**Discussion by shaburov** [allure-framework/allure2#2212] [TestOps]
@cheshi-mantu I can't follow your [recommendation](https://github.com/orgs/allure-framework/discussions/2186#discussioncomment-7501998)

`For he future reference, we support Allure Testops via dedicated support portal https://help.qameta.io/, all bugs, enhancement suggestions are to be reported via portal, this is the only way we can react in a timely manner.`

## Chrome

![285522158-b2a97ee3-158c-4885-abe2-31af0490581e](https://github.com/allure-framework/allure2/assets/2149631/6fca1d89-bc07-423d-85b6-0500237eeec2)

## Safari

![285522201-0cfe09e4-ca80-484a-bd39-eaeeb3ecf2a5](https://github.com/allure-framework/allure2/assets/2149631/c11a7d5c-acc4-4c7e-9430-8d68a125a009)

**Comment by cheshi-mantu** [comment]
here we can't help directly. Chrome and other browsers with installed addons can experience this issue. I'm periodically checking this (you are not the first one) behaviour on chromium based browsers and Safari, and I'm still having 100% success rate with incognito mode or all addons disabled. Sometimes the behaviour is fixed just by updating to the most recent Chrome version.

**Reply by cheshi-mantu** [reply]
alternatively you can send an email to help qameta.io email address and your account will be created automatically and you will receive then the activation email.
Meanwhile we'll check this behaviour with the help desk vendor.

**Reply by cheshi-mantu** [reply]
seems to be fixed 
![image](https://github.com/allure-framework/allure2/assets/8607348/2b6b6c06-54b8-4c84-80b0-5a9e2864a53e)
